go-controller/pkg/node/gateway_init.go

```diff
@@ -198,13 +198,19 @@ func (n *OvnNode) initGateway(subnets []*net.IPNet, nodeAnnotator kube.Annotator
 		klog.Info("Preparing Shared Gateway")
 		gw, err = newSharedGateway(n.name, subnets, gatewayNextHops, gatewayIntf, nodeAnnotator)
 	case config.GatewayModeDisabled:
+		var chassisID string
 		klog.Info("Gateway Mode is disabled")
 		gw = &gateway{
 			initFunc:  func() error { return nil },
 			readyFunc: func() (bool, error) { return true, nil },
 		}
+		chassisID, err = util.GetNodeChassisID()
```

Comment:
@cathy-zhou this could just be `chassisID, err := ...` since Go actually lets you redefine `err` here even if already defined. Go is odd that way.

Reply:
With that, make lint shows:
```
pkg/node/gateway_init.go:211:3: ineffectual assignment to `err` (ineffassign)
                err = util.SetL3GatewayConfig(nodeAnnotator, &util.L3GatewayConfig{
                ^
```
and this err return value is not used outside the switch code block.